cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraRDDWriter.scala

```diff
@@ -35,15 +41,28 @@ import java.util.concurrent.Executors
 
 import scala.collection.JavaConversions._
 
+
 object CassandraRDDWriter {
 
   def write[K: AvroRecordCodec, V: AvroRecordCodec](
+    rdd: RDD[(K, V)],
+    instance: CassandraInstance,
+    layerId: LayerId,
+    decomposeKey: K => Long,
+    keyspace: String,
+    table: String,
+    threads: Int = ConfigFactory.load().getThreads("geotrellis.cassandra.threads.rdd.write")
```

Comment:
I suggest pulling the default number of threads into a final (constant) member of the object.